### ISSUE TITLE
Better "Prettify" behavior for longer graphql code (more line breaks!)

### ISSUE THREAD

**Issue by m4ttheweric** [hasura/graphql-engine#8166]
### Is your proposal related to a problem?

When designing a query/mutation/subscription in the API Explorer, if you have a lot of fields or variables, the code gets unruly pretty fast. If I paste it into VSCode and let Prettier do it's thing, then it vastly improves the readability.

Can this be adjusted to work on par with Prettier or better yet just use Prettier?

I have example photos here...

Ugly (current behavior):
![Screen Shot 2022-02-09 at 11 57 17 AM](https://user-images.githubusercontent.com/49927862/153261932-e2fe448c-bcf4-4a5e-bfaf-3b7f2dec8570.png)

Pretty(er) (using VS Code and Prettier extension):
![Screen Shot 2022-02-09 at 11 57 39 AM](https://user-images.githubusercontent.com/49927862/153261936-17bbe559-8b78-4813-bd0f-e599a5300f13.png)

**Comment by abdullah2993** [comment]
Yeah, this would be a very nice to have feature even better if we have a little more control over how it looks, atm we have to resort to https://jsonformatter.org/graphql-formatter to format the query.
